app/services/permissions/order.rb

```diff
@@ -43,13 +45,13 @@ def visible_line_items
 
     # Any line items that I can edit
     def editable_line_items
-      if @user.can_manage_line_items_in_orders_only?
-        Spree::LineItem.editable_by_producers(
-          @permissions.managed_enterprises.select("enterprises.id")
+      Spree::LineItem.editable_by_producers(
+        @permissions.managed_enterprises.select("enterprises.id")
+      ).or(
+        Spree::LineItem.where(
+          order_id: filtered_orders(managed_or_coordinated_orders_where_clause).select(:id)
         )
-      else
-        Spree::LineItem.where(order_id: editable_orders.select(:id))
-      end
+      )
     end
```

Comment:
Nice to see less code!

I can see how this is potentially more efficient, removing the need to check `can_manage_line_items_in_orders_only?`. If so, it would be nice to show this with a `query_database` spec if possible.

But I'm curious, was there a need to change it?